docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,46 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+You can define custom `tsh` command aliases using the `aliases` section. It also allows for changing of the default flag values for existing commands.
+
+New command `tsh l`:
+
+```yaml
+aliases:
+    "l": "tsh login --auth=okta --username=alice"
```

Comment:
Should not this and other examples use `$TSH` instead of `tsh`? To make sure that the same `tsh` is being invoked and not necessarily the one in PATH.

Reply:
Most users will only have a single `tsh` around. To that end `$TSH` is confusing and making the UX worse. I can see it being useful for Teleport developers, but that is a rather niche group compared to the rest of users?

Reply:
Thinking about it further, I guess we can add it as a hidden feature?

Reply:
Done: https://github.com/gravitational/teleport/pull/14931/commits/54dd074cf7c33afcbc90de70a0bbf137f025e810

Reply:
I don't even think it should be hidden TBH, I would make it a default actually and document here. I'm not sure why it would be confusing, it's basically environment variable expansion which users should be familiar with. Otherwise people may be shooting themselves in the foot, I imagine someone just downloading tsh and not installing it in PATH and then aliases that just use "tsh" will not work (while `$TSH` will).

Reply:
Makes sense, I've added docs for `$TSH` along with the rationale.